docs/docs/_partial-autologin-changes.mdx

```diff
@@ -0,0 +1,15 @@
+<details>
+<summary>LANGFLOW_AUTO_LOGIN and LANGFLOW_SKIP_AUTH_AUTO_LOGIN options</summary>
+
+In Langflow versions earlier than 1.5, if `LANGFLOW_AUTO_LOGIN=True`, then Langflow automatically logged users in as a superuser without requiring authentication.
+In this case, API requests didn't require a Langflow API key.
+
+In Langflow version 1.5, you could set `LANGFLOW_SKIP_AUTH_AUTO_LOGIN=True` _and_ `LANGFLOW_AUTO_LOGIN=True` to skip authentication for API requests _and_ allow automatic login as a superuser for all users.
+This was a temporary bypass for backwards compatibility.
+
+In Langflow version 1.6 and later, `LANGFLOW_AUTO_LOGIN=True` and `LANGFLOW_SKIP_AUTH_AUTO_LOGIN=False` is the default behavior. This is a **breaking change** introduced in v1.6.
+API Keys are now required to authenticate requests when `LANGFLOW_AUTO_LOGIN=True` _and_ `LANGFLOW_SKIP_AUTH_AUTO_LOGIN=False`.
+You may revert to prior behavior by setting `LANGFLOW_SKIP_AUTH_AUTO_LOGIN=True`.
```

Comment:
The `LANGFLOW_SKIP_AUTH_AUTO_LOGIN` environment variable will be removed in a future release.